FogCreek/Glitch-Community
# Profuse tea: Don't write cms data to the file system

## Links
https://profuse-tea.glitch.me/
https://app.clubhouse.io/glitch/story/4593/prismic-handling-research-implementation

## Changes:
- The home data and pupdates get loaded form the cms on startup, and new updates cause a reload
- Content update requests fall back on the last good data rather than using a generic fallback

## How To Test:
- Try loading the site and make sure all of the content is there
- Throw some garbage code in one of the getters and watch the site not explode or lose content

## Feedback I'm looking for:
- This still expects the curated home/pupdates files to be there, even if the aren't used outside of failure cases. We need some way for them to be updated, either as part of the build process or in the prismic webhook.
- The api updaters discard the POST content and reload from the cms. Do they still need the secret/security?
